backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -138,10 +138,14 @@ func toProto(t *time.Time) (*timestamp.Timestamp, error) {
 
 func (fs *experimentStore) StopExperiments(ctx context.Context, ids []uint64) error {
 	if len(ids) != 1 {
-		// TODO: This API will be renamed to StopExperiment and will take a single ID as a parameter
+		// TODO: This API will be changed to take a single ID as a parameter
 		return errors.New("A single ID must be provided")
 	}
-	sql := `DELETE FROM experiment_run WHERE id = $1`
+	sql :=
+		`UPDATE experiment_run 
+         SET execution_time = tstzrange(lower(execution_time), NOW(), '[]') 
```

Comment:
does this update execution time only if:
1. end_time is nil
2. Or `end_time` > `Now()`

Asking mainly because of 2) which is needed if we don't want to allow people to stop already stopped experiment.

Reply:
I'll have to add an additional predicate to the where clause to support this: 'AND (end_time == nil OR end_time > Now())

Reply:
sounds good